src/sentry/static/sentry/app/components/search/sources/helpSource.jsx

```diff
@@ -130,11 +130,13 @@ function buildHit(hit, options) {
     htmlString: _highlightResult.title.value,
     markTags: HIGHLIGHT_TAGS,
   });
-  const description = parseHtmlMarks({
-    key: 'description',
-    htmlString: _snippetResult[descriptionKey].value,
-    markTags: HIGHLIGHT_TAGS,
-  });
+  const description = _snippetResult
+    ? parseHtmlMarks({
+        key: 'description',
+        htmlString: _snippetResult[descriptionKey].value,
+        markTags: HIGHLIGHT_TAGS,
+      })
+    : {};
```

Comment:
Is it ok for description to be `undefined`?

Reply:
It's an object, but `description.value` will be undefined. It seems happy enough to render nothing.